ts/private/ts_project_worker.js

```diff
@@ -29,85 +29,138 @@ function getArgsFromParamFile() {
 function createFilesystemTree(root, inputs) {
     const hashes = new Map();
     const tree = {};
+    const watchingTree = {};
+
+    const TYPE = {
```

Comment:
Should we call this `NODE_TYPE`?

Reply:
Nodes represent the filesystem nodes so I am not sure whether renaming would make this more ambiguous. Also, since these are only attached to nodes, makes the prefix a bit redundant?

Reply:
I guess if it's always `node[TYPE.*]` it makes sense. Somewhere down below I saw `TYPE.*` and I had to scroll up to understand what was going on so I thought I'd mention it. You're probably right though 👍